src/scss/tools/functions/_units.scss

```diff
@@ -1,3 +1,5 @@
+@use 'strip-units' as functions;
```

Comment:
If we needed to import another function, would we need to change this to `as *`?

Reply:
I don't think so – I think you can import multiple @use into the same namespace, but it's probably something we'd need to test out

Reply:
Yup, I think going back to just using * might be useful as a default thing rather than giving it a namespace like this.

Reply:
The namespace is unique so will error if the same name is used. I have updated this to use the file name instead so this problem should go away now 👍